packages-web/maps/src/Maps.editorConfig.ts

```diff
@@ -56,13 +89,21 @@ export function getProperties(values: MapsPreviewProps, defaultProperties: Prope
 export function check(values: MapsPreviewProps): Problem[] {
     const errors: Problem[] = [];
 
-    if (values.mapProvider !== "openStreet" && !values.apiKey) {
+    if (values.apiKey && values.apiKeyExp) {
+        errors.push({
```

Comment:
Since we don't show both values next to each other, it will be hard for users to fix. Isn't it?